### ISSUE TITLE
Other models

### ISSUE THREAD

**Issue by paulrevival** [maxmati/a4tech-bloody-linux-driver#4]
Hey! How do you think, can I use it with my bloody R3 ?? And what does this driver do generally? :)

**Comment by maxmati** [comment]
I think that it should work with whole bloody series.
Right now it only allows you to change backlight level.

**Comment by paulrevival** [comment]
oh :) just a backlight))) Didn't you try to install bloody v5 driver (from off.website) to wine? I've tried to do that but it just extracted an archive and nothing more happend((( I just want to control a battery level because if I connect a mouse with less then ~50% (didn't try with more percent) it starts blinking with it's bloody-hand logo until I connect it with usb-cable and charge a little((( I saved (on windows) a battery min.level to start blinking from to mouse's memory but linux(Ubuntu) doesn't detect that setting I think(( What do you think?

**Comment by maxmati** [comment]
No, I didn't try to install drivers on wine. I think that miscellaneous settings like backlight or min. battery level aren't stored in persistent memory.
PS. I think that finding how to set min battery level should be very hard since you know exactly when message is sent to mouse.

**Comment by paulrevival** [comment]
damn! My old little logitech m185 showed a battery level on the top panel near the notebook battery level((( thanks for thoughts :) I just have no one contact (with wireless bloody mouse) to share my thoughts with :)
